src/java_bytecode/java_string_library_preprocess.cpp

```diff
@@ -287,32 +287,26 @@ exprt::operandst java_string_library_preprocesst::process_parameters(
   return process_operands(ops, loc, symbol_table, init_code);
 }
 
-/// Creates a string_exprt from the input exprt and adds it to processed_ops
-/// \param processed_ops: the list of processed operands to populate
-/// \param op_to_process: a list of expressions
+/// Creates a string_exprt from the input exprt representing a char sequence
+/// \param op_to_process: an operand of a type which implements char sequence
```

Comment:
Why are we still talking about operands here? Can't we name it `expr_to_convert` instead of `op_to_process`? And change the documentation to `an exprt of a type which implements char sequence`?